static/styles/zulip.css

```diff
@@ -2481,3 +2481,29 @@ button.topic_edit_cancel {
         margin-top: 10px;
     }
 }
+
+.nav-table a {
+    display: block;
+}
+.nav-table tr:hover {
+    text-decoration: none;
+    background-color: #eeeeee;
+    color: #0088cc;
+}
+.nav-table a:hover {
+    text-decoration: none;
+    color: #0088cc;
+}
+.nav-list-table a,
+.nav-list .nav-header {
+    text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
+}
+.ikon-style {
+    padding: 3px 10px;
+    margin-right: -38px;
+
+}
+.message-style {
+    padding: 3px 10px;
+    padding-left: 30px;
+}
```

Comment:
These CSS class names are too generic; they might overlap with CSS names we might use for other parts of the project!  Can you make sure all of them have `popover` in their name, at least?